tsfile/src/main/java/org/apache/iotdb/tsfile/fileSystem/FileInputFactory.java

```diff
@@ -19,33 +19,39 @@
 
 package org.apache.iotdb.tsfile.fileSystem;
 
+import java.io.IOException;
+import java.lang.reflect.InvocationTargetException;
+import java.nio.file.Paths;
 import org.apache.iotdb.tsfile.common.conf.TSFileDescriptor;
 import org.apache.iotdb.tsfile.read.reader.DefaultTsFileInput;
 import org.apache.iotdb.tsfile.read.reader.TsFileInput;
-import org.apache.iotdb.tsfile.write.writer.TsFileIOWriter;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import java.io.IOException;
-import java.nio.file.Paths;
-
 public enum FileInputFactory {
```

Comment:
Better to perform your refactoring on these two factories, too.

Reply:
Should I separate these different factories into three packages? like `fsFactory`, `fileInputFactory` and `fileOutputFactory`